### DISCUSSION TITLE
Copy & Paste files

### DISCUSSION THREAD

**Discussion by Nomex1337** [Ylianst/MeshCentral#5790] [General]
Hello Community,

i have installed Meshcentral on Synology/Docker 
 
when i connect with https://remote.mydomain.com and go to my agent devices an connect with one Meshagent Remote Desktop is all fine working.

But now if i copy a file and i want to paste this on my computer is no option to paste. I was even activating auto clipboard.

What i do wrong?   i clould not fine anythiong in the documentation.

Thnaks for Help

**Comment by si458** [comment]
Copy and paste is only for text. You can't copy and paste files sorry. You can activate the copy and paste by clicking settings and then checking auto clipboard and you must use chrome browser

**Comment by Nomex1337** [comment]
oh what a shame - why is it in teamviewer or anydesk working and not in meshcentral ?

**Reply by si458** [reply]
Because the it requires time to implement and also how would you paste a file on ur remote computer from inside a Web browser?
remember teamviewer/any desk are applications, where as meshcentral runs in ur Web browser! So u are limited to what a Web browser can do

**Reply by Nomex1337** [reply]
Ok Thanks. When there is no way right now then maybe in the future. But thnaks a lot my friend.
